presto-raptor/src/main/java/com/facebook/presto/raptor/storage/OrcStorageManager.java

```diff
@@ -180,6 +181,7 @@ public OrcStorageManager(
         this.shardRecorder = requireNonNull(shardRecorder, "shardRecorder is null");
         this.typeManager = requireNonNull(typeManager, "typeManager is null");
         this.deletionExecutor = newFixedThreadPool(deletionThreads, daemonThreadsNamed("raptor-delete-" + connectorId + "-%s"));
+        this.orcFileMetadataCodec = JsonCodec.jsonCodec(OrcFileMetadata.class);
```

Comment:
Bind it using the `jsonCodecBinder(binder).bindJsonCodec(OrcFileMetadata.class);`, and than inject it.